From b00b64f4d8cab721a40e097788aec6e099d5e1a0 Mon Sep 17 00:00:00 2001
From: Chris Zhong <zyw@rock-chips.com>
Date: Tue, 31 Oct 2017 16:59:15 +0800
Subject: [PATCH] ARM: dts: rk3066a-rayeager: add supports- for mmc

Add supports-emmc for emmc; supports-sd for sdcard; supports-sdio for
wifi.

Change-Id: I13d3918f41f63ed9b27e9969e6f89d1006c9d45c
Signed-off-by: Chris Zhong <zyw@rock-chips.com>
---
 arch/arm/boot/dts/rk3066a-rayeager.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/rk3066a-rayeager.dts b/arch/arm/boot/dts/rk3066a-rayeager.dts
index e9db01712e23..336782a6fbd2 100644
--- a/arch/arm/boot/dts/rk3066a-rayeager.dts
+++ b/arch/arm/boot/dts/rk3066a-rayeager.dts
@@ -173,6 +173,7 @@
 	bus-width = <8>;
 	cap-mmc-highspeed;
 	disable-wp;
+	supports-emmc;
 	non-removable;
 	num-slots = <1>;
 	pinctrl-names = "default";
@@ -326,6 +327,7 @@
 	bus-width = <4>;
 	disable-wp;
 	num-slots = <1>;
+	supports-sd;
 	pinctrl-names = "default";
 	pinctrl-0 = <&sd0_clk>, <&sd0_cmd>, <&sd0_cd>, <&sd0_bus4>;
 	vmmc-supply = <&vcc_sd>;
@@ -339,6 +341,7 @@
 	disable-wp;
 	non-removable;
 	num-slots = <1>;
+	supports-sdio;
 	pinctrl-names = "default";
 	pinctrl-0 = <&sd1_clk>, <&sd1_cmd>, <&sd1_bus4>;
 	vmmc-supply = <&vccio_wl>;
-- 
2.35.3

